src/TopologicalSort.php

```diff
@@ -19,47 +19,56 @@ class TopologicalSort extends BaseGraph
     /**
      * run algorithm and return an ordered/sorted set of Vertices
      *
-     * the topologic sorting may be non-unique depending on your edges. this
-     * algorithm tries to keep the order of vertices as added to the graph in
-     * this case.
+     * the topological sorting may be non-unique depending on your edges
```

Comment:
Was this removed intentionally? If so, should this be considered a BC break? Can you add some test cases for this?

Reply:
I don't think it should be considered BC - I'm not sure the topsort should not have made any potential promises about the ordering in the first place.

Reply:
> I'm not sure the topsort should not have made any potential promises about the ordering in the first place.

I agree that it should not have made this promise in the first place.

That being said, I suppose this is something *some* projects may currently rely on it. As such, I'm marking this as a BC break just in case and will merge this as-is.

Again, thank you! :+1: